spring-boot-project/spring-boot/src/main/java/org/springframework/boot/logging/Slf4JLoggingSystem.java

```diff
@@ -16,11 +16,14 @@
 
 package org.springframework.boot.logging;
 
+import java.util.Collections;
+import java.util.List;
 import java.util.logging.ConsoleHandler;
 import java.util.logging.Handler;
 import java.util.logging.LogManager;
 import java.util.logging.Logger;
 
+import org.apache.logging.log4j.jul.Log4jBridgeHandler;
```

Comment:
The SLF4J logging system should know anything that’s Log4j2- or Logback-specific. If there’s a need for the two logging system to use different bridge handlers then this class should be reworked and the handler logic pushed down into the two system-specific sub-classes.

Reply:
I hesitated on changing this class name to `JulBridgingLoggingSystem`, but that would break API compatibility.

I'll refactor the code and push the changes down to the `Log4j2LoggingSystem`.